support/Pipelines/Bbh/InitialData.py

```diff
@@ -196,6 +198,9 @@ def generate_id(
         )
     if pipeline_dir and not run_dir:
         run_dir = pipeline_dir / "001_InitialData"
+    if control:
+        run_dir = f"{run_dir}/ControlParams00"
```

Comment:
Please add an underscore for consistency with pipeline steps and segments directory names: `ControlParams_00`

Reply:
I've also changed to 3 0-padding for the same reason.